lbz/request.py

```diff
@@ -75,8 +75,9 @@ def json_body(self) -> Optional[dict]:
                 try:
                     self._json_body = json.loads(self.raw_body)
                 except ValueError as error:
-                    msg = f"The provided payload is invalid.\nPayload body:\n{self.raw_body}"
-                    raise BadRequestError(msg) from error
+                    raise BadRequestError(
+                        "Payload is invalid.\nPayload body:\n {!r}".format(self.raw_body)
```

Comment:
missing definite article

Reply:
Also, the previous formatting was prettier 😉